go/vm/geth/geth.go

```diff
@@ -179,18 +182,22 @@ func (s *stateDbAdapter) CreateAccount(common.Address) {
 	// ignored: effect not needed in test environments
 }
 
-func (s *stateDbAdapter) SubBalance(common.Address, *big.Int) {
+func (s *stateDbAdapter) CreateContract(common.Address) {
+	// ignored: effect not needed in test environments
```

Comment:
panic here as well ?

Reply:
All the unimplemented functions without a `panic` are called during at least one test and should not terminate the program execution.